OWNERS

```diff
@@ -5,7 +5,6 @@ approvers:
 - mimowo
 
 reviewers:
-- denkensk
```

Comment:
I would suggest transferring to emeritus_reviewers to keep the recognition

Reply:
We already have `emeritus_approvers` section where denkensk is present. I believe it's even stronger recognition

Reply:
sure, I missed that, makes sense